src/common.jl

```diff
@@ -103,10 +105,60 @@ function plot_map!(ga::GeoAxis, gdf::DataFrame; geom_col=:geometry, color=nothin
 
     return nothing
 end
+
 function plot_map!(gdf::DataFrame; geom_col=:geometry, color=nothing)::Nothing
     return plot_map!(current_axis(), gdf; geom_col=geom_col, color=color)
 end
 
+function plot_map(gdf::Union{DataFrame,DataFrameRow}; geom_col::Symbol=:geometry, color_by::Symbol)
+    f = Figure(; size=(600, 900))
+    ga = f[1, 1] = GeoAxis(
+        f[1, 1];
+        dest="+proj=latlong +datum=WGS84",
+        xlabel="Longitude",
+        ylabel="Latitude",
+        xticklabelpad=15,
+        yticklabelpad=40,
+        xticklabelsize=10,
+        yticklabelsize=10,
+        aspect=AxisAspect(0.75),
+        xgridwidth=0.5,
+        ygridwidth=0.5,
+    )
```

Comment:
That double assignment isn't necessary I don't think:

```suggestion
    ga = GeoAxis(
        f[1, 1];
        dest="+proj=latlong +datum=WGS84",
        xlabel="Longitude",
        ylabel="Latitude",
        xticklabelpad=15,
        yticklabelpad=40,
        xticklabelsize=10,
        yticklabelsize=10,
        aspect=AxisAspect(0.75),
        xgridwidth=0.5,
        ygridwidth=0.5,
    )
```